src/Sulu/Bundle/TestBundle/Testing/TestUserProvider.php

```diff
@@ -163,9 +165,5 @@ public function supportsClass($class)
      */
     private function setCredentials(UserInterface $user)
```

Comment:
But then I should also delete this function completely :see_no_evil: Already on it!